bin/zCluster

```diff
@@ -665,12 +673,12 @@ if __name__ == '__main__':
     matches=c.fetchall()
     if len(matches) == 0:
         c.execute("""CREATE TABLE redshifts (name text, RADeg real, decDeg real, origRADeg real, origDecDeg real,
-                  offsetArcmin real, offsetMpc real, z real, delta real, errDelta real)""")
+                  offsetArcmin real, offsetMpc real, z real, delta real, errDelta real, CS real, A real)""")
     for obj in catalog:
         try:
             c.execute("INSERT INTO redshifts VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)", 
```

Comment:
Same as earlier - missing , ?, ? - i.e., number of inserted parameters doesn't match